packages/sdk/src/sdk/sdk.ts

```diff
@@ -100,9 +100,13 @@ function initByChain<Symbols extends AssetSymbol, ChainKeys extends ChainKey>(
                 ChainKeys
               >([configBuilder.moonChain.ws, config.source.ws]);
               const meta = foreignPolkadot.getMetadata();
-              const nativeAsset =
-                configBuilder.assets[meta.symbol] ||
-                createDummyAsset(meta.symbol);
+              const nativeAsset = [
+                ChainKey.Equilibrium,
```

Comment:
Equilibrium in Prod seems to have this correct, so why modify it? I think it should just be for EquilibriumAlphanet

Reply:
Looks like they just fixed it. A few our ago it was wrong :)